src/index.html

```diff
@@ -8,6 +8,107 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container star0">
+      <div class="star gap">
+
+      </div>
+      <div class="star gap">
+
+      </div>
+      <div class="star gap">
+
+      </div>
+      <div class="star gap">
+
+      </div>
+      <div class="star">
+
+      </div>
+    </div>
```

Comment:
Implement 6 blocks with stars class
Each block should have an extra class stars--0, stars--1, stars--2 ... stars--5 (one extra class per block)

Each star in a block should have a class stars__star and no extra classes or inline styles
```suggestion
<div class="stars stars--0">
    <span class="stars__star"></span>
    <span class="stars__star"></span>
    <span class="stars__star"></span>
    <span class="stars__star"></span>
    <span class="stars__star"></span>
  </div>
```